src/ILVerify/src/ILImporter.Verify.cs

```diff
@@ -407,10 +418,39 @@ void CheckIsValidBranchTarget(BasicBlock src, BasicBlock target)
                     else
                         VerificationError(VerifierError.BranchOutOfHandler);
                 }
-                return;
             }
         }
 
+        bool IsDirectChildRegion(BasicBlock enclosingBlock, BasicBlock enclosedBlock)
+        {
+            int enclosingRegionStart;
+            if (enclosingBlock.TryIndex == null)
+                enclosingRegionStart = enclosingBlock.StartOffset;
```

Comment:
Would it make sense to always finish at `enclosingBlock.StartOffset`, even when we have `enclosingBlock.TryIndex`?

Reply:
True, this makes sense and would speed up the process since start offset should always be smaller than TryOffset.